src/api.js

```diff
@@ -0,0 +1,33 @@
+const getTrendingAlbums = (setRecommendedData) => {
+    return fetch('https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&year=2023&country=US&page=1&per_page=5&sort=hot', {mode:'cors'})
+        .then(res => {
+        if (res.ok) {
+            return res.json()
+        }
+        throw new Error('Something went wrong')
+        })
+        .then(data => setRecommendedData(data))
+        .catch(err => console.error(err))
```

Comment:
These two lines for both functions could be moved to the App.js file to avoid passing down the setter function